srcpkgs/python3-matplotlib/files/README.voidlinux

```diff
@@ -11,3 +11,5 @@ The backend can be defined in $XDG_CONFIG_HOME/matplotlib/matplotlibrc
 
 Matplotlib’s LaTeX support requires a working LaTeX installation and
 ghostscript.
+
+NOTICE: /etc/matplotlibrc is no longer supported as configuration file
```

Comment:
I think we can just drop this message outright. The notice has been posted for years.

Reply:
I agree. Done.